snippets/python_snippets/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "breez-sdk examples"
+version = "0.2.7"
+description = ""
+authors = ["ruben beck <ruben.beck@pm.me>"]
+license = "MIT"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8"
+ruff = "^0.1.4"
+breez-sdk = { git = "https://github.com/cnixbtc/breez-sdk.git", branch = "pypi", subdirectory = "libs/sdk-bindings/bindings-python" }
```

Comment:
Do we need to use this fork? Could it not use our bindings from the main repo?

Reply:
I removed it since we use the temporary package from test pypi.